mne_bids/__init__.py

```diff
@@ -1,13 +1,10 @@
 """MNE software for easily interacting with BIDS compatible datasets."""
 
 __version__ = '0.5.dev0'
-
-
-from mne_bids.write import (write_raw_bids,  # noqa: E501 F401
-                            make_dataset_description, write_anat,  # noqa: F401
-                            get_anonymization_daysback,  # noqa: F401
-                            _stamp_to_dt, _get_anonymization_daysback)
-from mne_bids.read import (read_raw_bids, get_head_mri_trans,  # noqa: F401
-                           get_matched_empty_room)  # noqa: F401
-from mne_bids.utils import make_bids_folders, make_bids_basename  # noqa: F401
-from mne_bids import commands  # noqa: F401
+from mne_bids import commands
+from mne_bids.read import (get_head_mri_trans, read_raw_bids,
+                           get_matched_empty_room)
+from mne_bids.utils import (get_anonymization_daysback,
+                            make_bids_basename, make_bids_folders)
+from mne_bids.write import (make_dataset_description, write_anat,
+                            write_raw_bids)
```

Comment:
It's a noble goal to sort everything but you're making life harder for the reviewers.  I am tripping up on this diff. Can you make the diff "minimal"?

I don't think we'll be able to keep things alphabetic in the long term unless we make a policy like that but I think that would be a separate discussion.